doc/man1/openssl-ocsp.pod.in

```diff
@@ -24,10 +24,10 @@ B<openssl> B<ocsp>
 [B<-req_text>]
 [B<-resp_text>]
 [B<-text>]
-[B<-reqout> I<file>]
-[B<-respout> I<file>]
-[B<-reqin> I<file>]
-[B<-respin> I<file>]
+[B<-reqout> I<filename>]
```

Comment:
How about filepath?
```suggestion
[B<-reqout> I<filepath>]
```

Reply:
I agree that `filepath` would be better, 
but for some reason(s)`file` and `filename` are used throughout the OpenSSL man pages.